topics/mvvmcross/index.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases: monocross
+created_by: Alexandru Stefan Popescu
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
+topic: mvvmcross
+url: https://www.mvvmcross.com/
+---
+MvvmCross is a cross-platform MVVM framework that enables developers to create powerful cross platform apps. It supports Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP), and Windows Presentation Framework (WPF).
+
+The high level features that MvvmCross provides you with are:  
```

Comment:
Instead of this paragraph and the bulleted list below, can you describe the topic in sentences?

Reply:
Couldn't find another description and I'd like to not create one of my own and to keep the one from their website.

Reply:
I think creating your own description for topics is fine! The content provided for each topic is supposed to help those not familiar with the subject get acquainted, so summarizing what a topic is about by pulling from various data sources is a good thing.